main/zoe/api/zoe-contract-facet.md

```diff
@@ -349,14 +349,27 @@ zcf.shutdown();
 ## zcf.getTerms()
 - Returns: `{Object}`
 
-Returns the `issuers`, `brands`, and custom `terms` the current contract `instance` was instantiated with.
+Returns the `issuers`, `brands`, `maths`, and custom `terms` the current contract `instance` was instantiated with.
+
+The returned values look like:
+```js
+{ brands, issuers, maths, customTermA, customTermB ... }
+// where brands, issuers, and maths are all keywordRecords, like:
+
+{
+    brands: { A: moolaKit.brand, B: simoleanKit.brand },
+    issuers: { A: moolaKit.issuer, B: simoleanKit.issuer },
+    maths: { A: moolaKit.amountMath, B: simoleanKit.amountMath },
+    whatever: 'whatever',
```

Comment:
Hmm, can we use `customTermA` and `customTermB` instead of `whatever` to be consistent?

```suggestion
    customTermA: 'something',
    customTermB: 'something else',
```